doc-content/enterprise-only/openshift/freeform-server-immutable-deploy-assy.adoc

```diff
@@ -1,7 +1,8 @@
 [id='freeform-server-immutable-deploy-assy']
 = Deploying an immutable {KIE_SERVER} instance for a freeform environment
```

Comment:
Should we remove the wording 'instance for a freeform environment' consistently?